src/main/java/com/notably/model/block/BlockModel.java

```diff
@@ -0,0 +1,61 @@
+package com.notably.model.block;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * API of the BlockModel component.
+ */
+public interface BlockModel {
+    /**
+     * Gets the title of a block.
+     */
+    Title getTitle();
+
+    /**
+     * Gets the body content of a block.
+     */
+    Body getBody();
+
+    /**
+     * Gets the parent block of a block, if possible.
+     * Returns an {@code Optional<Empty>} if the block is the root block.
```

Comment:
Nit: I believe `Optional<Empty>` should be `Optional.empty()` instead?

Reply:
Will fix!